go.mod

```diff
@@ -15,6 +15,7 @@ require (
 	k8s.io/code-generator v0.24.3
 	k8s.io/component-base v0.24.3
 	k8s.io/klog/v2 v2.70.1
+	k8s.io/kube-openapi v0.0.0-20220328201542-3ee0da9b0b42
```

Comment:
When I apply `go mod tidy` to this commit, it undoes the change wrt kube-openapi.  That is, `go mod tidy` puts it back the way @mra-ruiz had it.

Reply:
hmm, I actually tested `go mod tidy` yesterday, before submitting this PR to make sure that the command doesn't introduce changes.

Just now I tried another thing. I manually add `//indirect` (which is the only difference I see) for `k8s.io/kube-openapi v0.0.0-20220328201542-3ee0da9b0b42`, then a `go mod tidy` still insists it is 'direct' by removing that comment.

What could cause the different behaviors? My `go version` shows `go version go1.19.4 darwin/amd64`.

Reply:
Aha, I found the cause. I have some extra code which is not added to git yet. That part of code directly depends on the module. Let me adjust the PR.